### PR TITLE
Add more well-known attributes

### PR DESCRIPTION
**Pull request by realloc**
Resolves nspcc-dev/neofs-api-go#172

### CODE REVIEW

File: object/types.proto [hunk 1]
```diff
@@ -87,8 +87,19 @@ message Header {
...
+  //   Marks smaller parts of a split bigger object
+  // * __NEOFS__EXPIRATION_EPOCH \
+  //   Tells GC to delete object after that epoch
+  //
+  // And some well-known attributes used by applications only:
+  //
+  // * Name \
+  //   Human-friendly name
+  // * FileName \
+  //   File name to be associated with the object on saving
```
Comment (alexvanin) [inline object/types.proto:100]: Not sure about having both `Name` and `FileName`. What's the difference then?

File: container/types.proto [hunk 2]
```diff
@@ -33,9 +33,16 @@ message Container {
...
-  // * Subnet \
+  // * __NEOFS__SUBNET \
   //   String ID of container's storage subnet. Container can be attached to
   //   only one subnet.
+  //
+  // And some well-known attributes used by applications only:
+  //
+  // * Name \
+  //   Human-friendly name
+  // * LocalCreationUnixTime \
```
Comment (pkorotkov) [inline container/types.proto:44]: Why local in `LocalCreationUnixTime`? I believe UTC timestamps are more appropriate for that.

File: object/types.proto (hunk 1 above)
Comment (alexvanin) [reply object/types.proto:100]: File can be split for multiple objects and each split object can have some unique custom text identifier independent from original file name. Got it.

File: container/types.proto (hunk 2 above)
Comment (realloc) [reply container/types.proto:44]: Regular UTC timestamp may be encoded in lots of different formats (ISO-8601, RFC 1123, etc) while UnixTime (still in UTC) in string presentation can be encoded consistently.

Comment (pkorotkov) [reply container/types.proto:44]: Well, the question is whether the value is in local time or UTC. If this is in UTC, let's put the matter to rest.

Comment (realloc) [reply container/types.proto:44]: I agree that the name is misleading. It should reflect that this timestamp is created on user side without verification and may have any timestamp not related to reality.

Does anybody have suggestions on how to name it in a better way?

Comment (pkorotkov) [reply container/types.proto:44]: I'd suggest not to bother with super-unombigous naming here as we simply cannot control it. A user can technically use our "well-known" attributes as they please, e.g. may write string `My birthday` to `LocalCreationUnixTime` and we must expect it to happen. Names of attributes are indicative and this one might be just `Timestamp`, for instance.

**Review by alexvanin — DISMISSED** [review]

**Review by pkorotkov — APPROVED** [review]

**Pull request merged**